packages/state-transition/test/utils/testFileCache.ts

```diff
@@ -103,12 +102,13 @@ async function downloadTestFile(fileId: string): Promise<Buffer> {
   const fileUrl = `${TEST_FILES_BASE_URL}/${fileId}`;
   // eslint-disable-next-line no-console
   console.log(`Downloading file ${fileUrl}`);
-
-  const res = await got(fileUrl, {responseType: "buffer"}).catch((e: Error) => {
-    e.message = `Error downloading ${fileUrl}: ${e.message}`;
-    throw e;
-  });
-  return res.body;
+  try {
```

Comment:
Any reason we can't follow a similar structure as before?
Also, the same construct as before can be used to remove the `Buffer` use.